bazel/repositories.bzl

```diff
@@ -854,6 +854,7 @@ def _com_github_google_quiche():
         name = "com_github_google_quiche",
         genrule_cmd_file = "@envoy//bazel/external:quiche.genrule_cmd",
         build_file = "@envoy//bazel/external:quiche.BUILD",
+        patches = ["@envoy//bazel/external:quiche.patch"],
```

Comment:
Please add a `TODO(birenroy):...` comment for the patch explaining when and how the patch will be removed. Thanks

Reply:
Done, thanks.